### ISSUE TITLE
update documentation for PASDA workflow

### ISSUE THREAD

**Issue by karenmajewicz** [geobtaa/metadata#132]

**Comment by karenmajewicz** [comment]
Updated scripts here: https://github.com/geobtaa/workflows/tree/main/harvesting/08a-01_pasda


- [ ] run datasetURLs.py
- [ ] run html2csv.py
- [ ] Fill in Resource Class (most will be `Datasets`.  Do a keyword search for Imagery or aerial on the title and fill those in with `Imagery`
- [ ] Date Range: Use the temporal coverage to create a value of `yyyy-yyyy`
- [ ] Upload these records to GEOMG as drafts
- [ ] Delete the extraneous characters in the INFORMATION field and the ID field. They are enclosed by `['` and/or `']` which need to be removed.

**Comment by karenmajewicz** [comment]
I am still working on parsing the Metadata files for the bounding boxes. TBD.

**Comment by karenmajewicz** [comment]
This script and workflow does a few things differently than we had before:

- [ ] creates IDs that are pasda-{datasetnumber} instead of creating UUIDs for them. This will tie the record more closely to PASDA and hopefully make it easier to reaccession in the future.
- [ ] programmatically fills in the default fields

**Comment by karenmajewicz** [comment]
Important fields we are missing from this process include:

- Spatial Coverage: some of this could be scripted, but will still need manual cleanup

- Bounding Boxes: this information is only present in the supplemental metadata found at the "Metadata" links in the portal. These files are presented within a .NET framework and are a little challenging to parse. Even though the underlying file is a known ISO or FGDC XML, the link from the portal shows an HTML type file. We used to have a link to a web folder shoring the original XMLs, but that seems to have been moved.
